constants/constants.go

```diff
@@ -0,0 +1,3 @@
+package constants
```

Comment:
Yep we should try to use it for all our constants, like list of supported envs, specific commands, etc. If we don't find more uses, we can always get rid of it